README.md

```diff
@@ -1,63 +1,71 @@
-GCP Structure Tests
+Container Structure Tests
 ====================
 
-The GCP Structure Tests provide a powerful framework to validate the structure 
-of a container image. These tests can be used to check the output of commands 
+The Container Structure Tests provide a powerful framework to validate the structure
+of a container image. These tests can be used to check the output of commands
 in an image, as well as verify metadata and contents of the filesystem.
 
-Tests can be run either through a standalone binary, or through a Docker image as part of a cloudbuild. Download the binary
-[here](https://storage.googleapis.com/container-structure-test/latest/container-structure-test),
+Tests can be run either through a standalone binary, or through a Docker image.
+
+## Installation
+Download the latest binary release [here](https://storage.googleapis.com/container-structure-test/latest/container-structure-test),
 or pull the image at `gcr.io/gcp-runtimes/container-structure-test`.
 **Please note that at this time the binary is only compatible with Linux.**
 
+## Container Structure Test Workflow
+To use container structure tests to validate your containers, you need the following:
+- The container structure test binary or docker image
+- A container image to test against
+- A test .yaml file with user defined structure tests to run inside of the specified container image
```

Comment:
technically this can be a yaml or json file

Reply:
fixed